### ISSUE TITLE
[Bug]: 3 matches, excluded 2 => No comments left to remove!

### ISSUE THREAD

**Issue by NatNgs** [ThioJoe/YT-Spammer-Purge#592]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

Run a scan on a few videos, matching no Spammers, but matching 3 "Commenters, But Who Wrote Many Similar Comments".

Excluding 2 false-positives (still one should be reported), and returned to me that I've excluded all, so app will stop (see logs)

### Release version

2.15.0

### Steps to reproduce

1. Mode `1. Scan specific videos`
2. Added a list (coma separated) of ~30videos, like this:
> `https://www.youtube.com/watch?v=VnY8QQKRbro,https://www.youtube.com/watch?v=uaNFvORh-9c,https://...`
3. Is this video list correct? (y/n): `y`
4. Mode `1. (RECOMMENDED): Auto-Smart Mode: Automatically detects multiple spammer techniques`
5. Scanning...
``` Total: [35249/35249] (100%) | Comments Scanned: 24248 | Replies Scanned: 11001 | Matches Found So Far: 0
Number of Matched Comments Found:  0
Number of Non-Matched But Duplicate Comments Found:  65
```
6. Input: exclude all but one duplicate comment found

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
============================ Match Samples: One comment per matched-comment author ============================


-------------------------  Non-Matched  Commenters, But Who Wrote Many Similar Comments -------------------------
---------------------------- ( Similarity Threshold: 90%  |  Minimum Duplicates: 4 ) ----------------------------
1.  [x26]  Oscar O'sullivan    : This should help https://youtu.be/48Dc7bqU_Dg thank God chip pans are largely a...
2.  [x19]  Randy Carpadus      : There are some out there. Usually jealous incels.
3.  [x20]  New York Yankees    : 😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍�😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😍😚�😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚😚...

==================== (See log file for channel IDs of matched authors above) ====================

 NOTE:  Check that all comments listed above are indeed spam.
 > If you see missed spam or false positives, you can submit a filter suggestion here: TJoe.io/filter-feedback

How do you want to handle the matched comments above?
~~ Not Your Channel Mode: Only Reporting is Possible ~~
 > To exclude certain authors: Type 'exclude' followed by a list of the numbers (or ranges of #'s) from the sample list
      > Example:  exclude 1, 3-5, 7, 12-15
 > To only process certain authors: Type 'only' followed by a list of the numbers (or ranges of #s) from the sample list
      > Example:  only 1, 3-5, 7, 12-15  --  (Will effectively exclude the 'inverse' of the 'only' selected authors)
 > To report the comments for spam, type 'REPORT'.
 > To do nothing and only log, type 'NONE'

 (Not Case Sensitive) Input: exclude 1,2

All 45 comments from the following users are now excluded from deletion:
    User ID: UCp3PZIxUp582_g8lPtYmmwg   |   User Name: Oscar O'sullivan
    User ID: UCI9eN7F9ZgzZdr6nZWXLOMQ   |   User Name: Randy Carpadus


Add these excluded users to the whitelist for future scans?

Whitelist Users? (y/n): y

Press Enter to decide what to do with the rest...

All authors excluded, no comments left to remove!

Press Enter to log and/or return to main menu...
```

### Screenshots

![image](https://user-images.githubusercontent.com/10955538/152841819-92bd0282-178c-44fd-b1b5-25f5db8d0d16.png)

**Comment by ThioJoe** [comment]
hmm yea that doesn't seem right

**Comment by ThioJoe** [comment]
Can you link to the videos that had the first two results? That way I can replicate it. The links to the comments would be in the log file.

**Comment by NatNgs** [comment]
Videos codes are: UCI9eN7F9ZgzZdr6nZWXLOMQ & UCp3PZIxUp582_g8lPtYmmwg
RTF file if needed: https://pastebin.com/i6zdsCRE

**Comment by ThioJoe** [comment]
Ah those look like channel IDs actually, video IDs are the ones after the "watch?v=" and are 11 characters long. Also looks like pastebin removed the file for some reason. You should be able to drag the file right into the github comment box though.

**Comment by NatNgs** [comment]
Strange, these codes does not seems to be channelId neither...

Also, didn't worked to push it to github in rtf format, renamed as ".txt" below:

[Spam_Log_2022-02-07_18-15-58.txt](https://github.com/ThioJoe/YT-Spammer-Purge/files/8018797/Spam_Log_2022-02-07_18-15-58.txt)

Hope you have all information needed (I didn't take record of what videos I've pushed in this scanned list)

**Comment by ThioJoe** [comment]
Ok should be fixed with the new release, 2.15.1
